src/groupeddataframe/grouping.jl

```diff
@@ -156,10 +190,12 @@ Compat.lastindex(gd::GroupedDataFrame) = length(gd.starts)
 Base.first(gd::GroupedDataFrame) = gd[1]
 Base.last(gd::GroupedDataFrame) = gd[end]
 
+# Single integer indexing
 Base.getindex(gd::GroupedDataFrame, idx::Integer) =
     view(gd.parent, gd.idx[gd.starts[idx]:gd.ends[idx]], :)
 
-function Base.getindex(gd::GroupedDataFrame, idxs::AbstractArray)
+# Array of integers
+function Base.getindex(gd::GroupedDataFrame, idxs::AbstractArray{T}) where {T<:Integer}
```

Comment:
following my comment above
```suggestion
function Base.getindex(gd::GroupedDataFrame, idxs::Union{AbstractVector{T}, Not}) where {T<:Integer}
```